go/merkletree/merkle_verifier_test.go

```diff
@@ -1,10 +1,13 @@
 package merkletree
 
 import (
+	".."
```

Comment:
I'd prefer the absolute one (I believe that's what is recommended in Effective Go?).